central/image/service/service_impl.go

```diff
@@ -86,6 +88,8 @@ var (
 	reprocessInterval = env.ReprocessInterval.DurationSetting()
 
 	delegateScanPermissions = []string{"Image"}
+
+	imageScanMetricsName = prometheus.Labels{"location": "image-scan"}
```

Comment:
Why did you choose this label convention? It's not really clear to me what `location` means in this context. Also given the `image_scan` prefix of the metric, a label value of `image-scan` seems redundant. What are you trying to communicate? I think there is probably a better name.

Reply:
The `location` would be where the scan semaphore is located, since we have  multiple different ones (in image scan, image enricher, and local scan). Agree with the prefix of the metric, it should just be scan probably, renamed it to that. If you have a better idea than `location` I'm up for it.


Regarding the other comment I've done plenty of testing and some improvement of the metrics and I think the data is correct now.
(Sorry for the late response, I was debugging the metrics and also occupied with something else.)

Reply:
I'm not a native speaker, but afaik `location` suggests a physical location - for example, I'm located in Germany. That doesn't make sense in this context about scanners. Also I think the `image_scan` prefix is fine, but not as a label value. I'd suggest something like `entity={central-service|enricher|sensor}`.